### DISCUSSION TITLE
Cannot get authentication

### DISCUSSION THREAD

**Discussion by Lasershane7** [homebridge-simplisafe3/homebridge-simplisafe3#399] [General]
Clicking authenticate bring me to a blank SimpliSafe page and seems to be stuck on browser detection. This is on Mac and PC as well as multiple browsers. 

Stuck URL below. 

https://auth.simplisafe.com/u/mfa-detect-browser-capabilities?state=hKFo2SBna1h0dHJ2Yk9WX1djWjV2VHlycGNMeDE4ZndXSElUNqFuqm1mYS1kZXRlY3SjdGlk2SByS1ZVaUI5TFlqajd2NEN2eDJwT0ZDWGVadFI0OElFdaNjaWTZIDQyYUJaNWxZclZXMTJqZk91dTNDUVJPaXR3eGc5c041

**Comment by shamoon** [comment]
Perhaps try on a different network? Generally speaking I have found Chrome the best to use for this.

**Reply by Lasershane7** [reply]
Done and done.
